sv_SE/LC_MESSAGES/unreadCommentsEmail.po

```diff
@@ -1,20 +1,20 @@
 msgid "EMAIL_SUBJECT_FORMAT"
-msgstr "[Sudomemo] %s, you've got new comments on your Flipnotes!"
+msgstr "[Sudomemo] %s, du har fått nya kommentarer på dina Flipnotes!"
 
 msgid "TITLE_NEW_COMMENTS"
-msgstr "New Comments"
+msgstr "Nya kommentarer"
 
 msgid "HELLO_FORMAT"
-msgstr "Hello, %s!"
+msgstr "hej, %s!"
 
 msgid "TEXT_YOU_HAVE_X_COMMENTS_FROM_FORMAT"
-msgstr "You have %d new comment(s) on Sudomemo from %s"
+msgstr "Du har %d nya kommentarer på Sudomemo från %s!"
```

Comment:
Please leave the extra exclamation point out, thanks -- this should not end with any punctuation.

Reply:
I also made minor corrections.